brand/models/embrace_campaign.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class EmbraceCampaign(models.Model):
```

Comment:
This certainly needs a validator to make sure that the configuration JSON is how you expect it to be. Checkout how `Brand` calls a function, `validate_tag`

What fields are required? "System Prompt" and "User Prompt" I'm assuming. Anything else?